chcemvediet/apps/obligees/tests/test_views.py

```diff
@@ -76,64 +76,143 @@ def test_allowed_http_methods(self):
         allowed = [u'HEAD', u'GET']
         self.assert_allowed_http_methods(allowed, reverse(u'obligees:autocomplete'))
 
-    @unittest.skip(u'FIXME')
     def test_autocomplete_returns_json_with_correct_structure(self):
-        oblg1 = self._create_obligee(name=u'Agency', street=u'Westend', city=u'Winterfield', zip=u'12345', emails=u'agency@a.com')
-        oblg2 = self._create_obligee(name=u'Ministry', street=u'Eastend', city=u'Springfield', zip=u'12345', emails=u'ministry@a.com')
+        neighbourhood1 = self._create_neighbourhood()
+        oblg1 = self._create_obligee(
+                official_name=u'Agency Official',
+                name=u'Agency',
+                name_genitive=u'Agency (G)',
+                name_dative=u'Agency (D)',
+                name_accusative=u'Agency (A)',
+                name_locative=u'Agency (L)',
+                name_instrumental=u'Agency (I)',
+                gender=Obligee.GENDERS.MASCULINE,
+                ico=u'12345678',
+                street=u'Westend',
+                city=u'Winterfield',
+                zip=u'12345',
+                iczsj=neighbourhood1,
```

Comment:
Kedze endpoint aj tak nevracia texty pouzite v `neighbourhood`, tak nemozeme tu rovno pouzit `self.neighbourhood`? A potom v asserte `self.neighbourhood.id`?

Analogicky pre `neighbourhood2`, tagy a groupy.

Reply:
Bude stacit jedna instancia v m2m relacii? Teda jeden `tag`.

Reply:
Hadam ano. Jedize ze by sme v buducnosti potrebovali test s vacsimi m2m relaciami. Potom to upravime, alebo spravime druhy test s vacsou m2m relaciou.